src/Symfony/Component/Translation/Tests/Catalogue/DiffOperationTest.php

```diff
@@ -35,7 +35,7 @@ public function testGetMessagesFromSingleDomain()
         );
 
         $this->assertEquals(
-            array('b' => 'old_b'),
+            array('a' => 'old_a'),
             $operation->getObsoleteMessages('messages')
```

Comment:
this is wrong. The obsolete key is ``b``, not ``a``. ``a`` is not obsolete as it is there in the new catalogue too.